src/platform/forms-system/src/js/fields/ObjectField.jsx

```diff
@@ -290,7 +290,8 @@ class ObjectField extends React.Component {
       </div>
     );
 
-    if (title) {
+    const renderFieldset = !!showFieldLabel;
+    if (title && !renderFieldset) {
```

Comment:
If this is still desirable, could we get a unit test for it?

Reply:
I think it's still something we'd want. I'll try and get a unit test in for it soon (on PTO later this week). Sorry this PR has been floating around out there for so long =\

Reply:
Added a couple unit tests for this condition

Reply:
I think I would prefer a new `ui:option` to force the ObjectField to return a `<div>` wrapper instead of a `<fieldset>`. We could test if the `properties` object is empty, but we'd still have cases where we end up with a nested `<fieldset>`.

Many times we only want to display content (e.g. text or an alert box), and if we use this minimal `schema`/`uiSchema` combo:

```js
// schema
'view:someText': {
  type: 'object',
  properties: {},
}

// uiSchema
'view:someText': {
  // if this was set as a plain string, we'd end up with a
  // fieldset + legend
  'ui:title': () => 'test',
}
```

We when up with a `<fieldset>`, without a `<legend>`. This then becomes an accessibility concern.

Resulting HTML:

```html
<div>
  <fieldset>
    <div class="schemaform-field-container schemaform-block">
      <div class="schemaform-block-header">test</div>
    </div>
  </fieldset>
</div>
```

Also, if you start messing with the other options that include setting the `ui:title` to a single space and some combo of `hideLabelText` and `showFieldLabel`, it duplicates the content!

[WAVE](https://wave.webaim.org/) tester reference:

- [1.1.1 Non-text Content (Level A)](https://webaim.org/standards/wcag/checklist#sc1.1.1)
- [1.3.1 Info and Relationships (Level A)](https://webaim.org/standards/wcag/checklist#sc1.3.1)
- [2.4.6 Headings and Labels (Level AA)](https://webaim.org/standards/wcag/checklist#sc2.4.6)
- [3.3.2 Labels or Instructions (Level A)](https://webaim.org/standards/wcag/checklist#sc3.3.2)

Reply:
This definitely might be the safer way to go 👍 I'll try and take a look at this option sometime this week, swamped with stuff before the Holiday right now though >.<